pkg/formatter/json.go

```diff
@@ -0,0 +1,22 @@
+package formatter
+
+import (
+	"encoding/json"
+)
+
+type jsonFormatter struct{}
+
+// NewJSONFormatter formats output into json
+func NewJSONFormatter() jsonFormatter {
+	return jsonFormatter{}
+}
+
+// Format returns the Marshalled json output
+func (p jsonFormatter) Format(data func() interface{}) (string, error) {
+	out, err := json.MarshalIndent(data(), "", "    ")
```

Comment:
Just a tiny nit, isn't json usually 2 indented. :)

Reply:
copied it from your package :trollface:

Reply:
Shit... :D :D :D :D